src/app/sc2/domain/slack.py

```diff
@@ -0,0 +1,115 @@
+""" Slack integration """
+import urllib
+
+from google.appengine.api import urlfetch
+
+from app.sc2.utils.match_utils import get_vs_string_from_match
+from app.sc2.utils.player_utils import prettify_name
+
+
+# TODO:slack hook this up when auth stuff figured out
```

Comment:
Addendum: `alert_match_closed` will send a message to the channel, this is for actually setting the channel topic